src/testRunners/jasmine/jasmine2.js

```diff
@@ -65,9 +67,20 @@ function jasmine2(config, environment, moduleLoader, testPath) {
               throw Error('lastCalledWith() should be used on a mock function');
             }
             const calls = actual.mock.calls;
-            const args = Array.prototype.slice.call(arguments, 1);
+            const expected = Array.prototype.slice.call(arguments, 1);
+            const actualValues = calls[calls.length - 1];
+            const pass = util.equals(actualValues, expected);
+            const message = [
+              'function wasn\'t called with the expected values.',
```

Comment:
based on `pass` you'll need to adjust the message to say `was called` or `wasn't called`. Please add a test for `expect(a).not.toBeCalled(…)` – it should have the inverted message :)